code/modules/surgery/surgery.dm

```diff
@@ -149,7 +149,7 @@
 	var/list/req_tech_surgeries = subtypesof(/datum/surgery)
 	for(var/i in req_tech_surgeries)
 		var/datum/surgery/beep = i
-		if(beep.requires_tech)
+		if(beep && istype(beep) && beep.requires_tech)
```

Comment:
this condition will never be true since subtypesof returns typepaths which always fail istype and always pass a lone truthiness test. The fix should be to change the condition to `initial(beep.requires_tech)` which is the only way to test for values off a typepath

Reply:
Thank